style.css

```diff
@@ -320,18 +326,10 @@ color: rgba(0, 0, 0, 0.46);
   padding-left: 4px;
 }
 
-.foot_section {
-  max-width: 1440px;
-}
 .foot_link {
-  text-decoration: none;
   color: #d9d8d6;
 }
-.footer_list {
-  display: flex;
-  list-style-type: none;
-  justify-content: flex-end;
-}
+
 
 .unordered_list_for_numbers {
```

Comment:
this part is not formatted. Remember to format code before push.
1. Enter VSCode settings
2. Type Prettier in searchbox
3. Check `format on save` checkbox
4. Forget about formatting 😉

Reply:
couldn't find this checkbox about prettier I have only 2 options via F1 It is either Create configuration file or reset module execution state. When I checked for options inside of prettier extension there is no such checkbox :)

Reply:
@sinkinshadow pardon me, it should be: `Editor:Format On Save` in settings. If Code asks you which formatter to use pick Prettier.

Reply:
got it :) TY